### ISSUE TITLE
Sync issue with snes/sgb cores ntsc and LG Flatron 21FB crt - Wobby picture

### ISSUE THREAD

**Issue by stainmage** [MiSTer-devel/SNES_MiSTer#371]
Hello,

I have specific sync issue with this CRT, it doesn't appear with other tv sets, nor with other cores with the same tv.

Waves go vertically acrosse the screen, wobbly picture. I don't understand why but it seems this tv is picky with this core only in NTSC mode.

I'm using Antonio Villena's scart with Mister Plus

Cheers

**Comment by birdybro** [comment]
Did you email @antoniovillena for support yet?

info@antoniovillena.es

He might know how to resolve this more easily than most.

**Comment by stainmage** [comment]
yes made an email, however we didn't find the culprit yet, I think it's a incompatibility between the sync signal of the snes core and the tv, still couldn't test with another cable but I'll do it later when I will reach the tv again

**Comment by birdybro** [comment]
What's the make and model of your CRT?

**Comment by stainmage** [comment]
LG Flatron RT-21FB75VX

**Comment by stainmage** [comment]
yeah I was suspecting that but why would it happen only with this core on ntsc ? all other cores I have tested on ntsc don't give this beahviour
